ansible/group_vars/all.yml

```diff
@@ -56,6 +56,9 @@ docker_proxy_image: "{{ docker_proxy_image_param | default('jupyterhub/configura
 ## Jupyter Notebook config
 jupyter_notebook_config: "{{ jupyter_notebook_config_param | default('jupyter_notebook_config.py') }}"
 
+# Admin users
+admin_user: "{{ admin_user_param | default('') }}"
```

Comment:
If this is correct we have to change the hosts.example to use the variable as plural

Reply:
I changed the item mentioned above, so will leave this setting as-is.